### DISCUSSION TITLE
Reply Keyboards not have action and parm is there a way to do that?

### DISCUSSION THREAD

**Discussion by 3bkry** [def-studio/telegraph#228] [Q&A] [answered]
Hi,
Please help me with this issue:
Reply Keyboards not have action and parm is there a way to do that?
![image](https://user-images.githubusercontent.com/14781002/195494829-c500a951-897b-4b6d-a1e6-2110647ede62.png)

Also is there a way to send a prefilled quiz?

**Comment by fabio-ivona** [comment, marked as answer]
Hi What kind of button are you trying to obtain?

Quizzes and Polls will be released today

**Reply by 3bkry** [reply]
Thank you @fabio-ivona I am waiting.
For ReplyButton could I call "action" and "parm" Because it`s only working on the normal Button only.

**Reply by fabio-ivona** [reply]
Telegram doesn't support actions for reply buttons
